File: Random Number/random.R
## 500 random numbers with normal distribution
rnorm(500)

#Random number with normal distribution using mean and standard deviation
rnorm(500,mean=10,sd=1);

# 500 random numbers between 0 and 100
runif(500, min=0, max=100);

#Random sample of 3 numbers in the range of 0 to 100 with replacement
sample(1:100, 3, replace=TRUE)

#visualize numbers distribution in a histogram
hist(rnorm(500));

#density
plot(density(runif(500, min=0, max=100)));

File: Random Number/random.cs
using System;

namespace RandomNumber
{
    class Program
    {
        static void Main(string[] args)
        {
            Random random = new Random();

            // Generate Random Number
            int num = random.Next();

            // Generate Number < 1000
            int numLess1000 = random.Next(1000);

            // Generate Number between a MIN & MAX Value
            int numMinMax = random.Next(10, 99);

            Console.WriteLine($" Random Number: {num} \n Less than 1000: {numLess1000} \n Min Max: {numMinMax} ");

            Console.ReadLine();
        }
    }
}
